l1-contracts/test/staking/deposit.t.sol

```diff
@@ -96,23 +112,20 @@ contract DepositTest is StakingBase {
     givenCallerHasSufficientFunds
     givenAttesterIsNotRegistered
   {
-    // it transfer funds from the caller
-    // it adds attester to the set
-    // it updates the operator info
-    // it emits a {Deposit} event
-
-    assertEq(stakingAsset.balanceOf(address(staking)), 0);
+    // it adds attester to the entry queue
+    // it emits a {ValidatorQueued} event
 
-    vm.expectEmit(true, true, true, true, address(staking.getGSE()));
-    emit IGSECore.Deposit(address(staking.getGSE().CANONICAL_MAGIC_ADDRESS()), ATTESTER, WITHDRAWER);
+    vm.expectEmit(true, true, true, true, address(staking));
+    emit IStakingCore.ValidatorQueued(ATTESTER, WITHDRAWER);
 
     staking.deposit({_attester: ATTESTER, _withdrawer: WITHDRAWER, _onCanonical: true});
-
-    assertEq(stakingAsset.balanceOf(address(staking.getGSE().getGovernance())), DEPOSIT_AMOUNT);
+    // the money is in the staking contract
+    assertEq(stakingAsset.balanceOf(address(staking)), DEPOSIT_AMOUNT);
+    // not in the GSE
+    assertEq(stakingAsset.balanceOf(address(staking.getGSE().getGovernance())), 0);
```

Comment:
This is not the gse that you are checking, you are checking the governance here. Might make sense to add the check for the `gse` explicitly as well.